cmd/finch/nerdctl_native.go

```diff
@@ -89,6 +87,73 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 
 var osAliasMap = map[string]string{}
 
-var osArgHandlerMap = map[string]map[string]argHandler{}
+var osArgHandlerMap = map[string]map[string]argHandler{
+	"container run": {
+		"--mount": handleBindMounts,
+	},
+}
 
 var osCommandHandlerMap = map[string]commandHandler{}
+
+func handleBindMounts(_ NerdctlCommandSystemDeps, _ *config.Finch, nerdctlCmdArgs []string, index int) error {
```

Comment:
Is it possible to combine the 3 separate `handleBindMounts` functions in the `_darwin` `_windows` and `_native` files? They seem similar

Reply:
Agreed, they are very similar.
I believe that there is an opportunity to move the `handleBindMounts` to the level of `nerdctl.go`, but the WSL path manipulation is the detail currently anchoring them at the OS level.

Reply:
There are two ways (which are roughly the same):
1. You could add a new function which encapsulates all of the OS specific logic (e.g. `handleBindmountPath`, or something), and does nothing on Linux (this is what I would prefer)
1. You could use runtime.GOOS to gate the logic

Reply:
Please refer to the [commit 6f4f775](https://github.com/runfinch/finch/commit/6f4f77561567c5d331d2f24dbf80c4fc98786d5a) as resolution to this comment.